### PR TITLE
Aligin ako 1.7.1 configurations

### PR DESCRIPTION
**Pull request by XudongLiuHarold**
**What this PR does / why we need it**:

- Align ako 1.7.1 configurations
- revert setting default service type to NodePortLocal, because we will achieve that in tanzu-framework repo: https://github.com/vmware-tanzu/tanzu-framework/pull/2938

**Which issue(s) this PR fixes**:

Fixes #

**Describe testing done for PR**:

**Special notes for your reviewer**:

**Release note**:

```release-note

```
**New PR Checklist**

- [x] Ensure PR contains only public links or terms
- [x] Use good commit [messages](https://github.com/vmware-tanzu/tanzu-framework/blob/main/CONTRIBUTING.md)
- [x] Squash the commits in this branch before merge to preserve our git history
- [ ] If this PR is just an idea or POC, use a [Draft PR](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests) instead of a full PR
- [ ] Add appropriate [labels](https://github.com/vmware-tanzu/load-balancer-operator-for-kubernetes/labels) according to what type of issue is being addressed.

### CODE REVIEW

File: api/v1alpha1/akodeploymentconfig_types.go [hunk 1]
```diff
@@ -143,16 +153,11 @@ type ExtraConfigs struct {
 
 	// ServicesAPI specifies if enables AKO in services API mode: https://kubernetes-sigs.github.io/service-apis/.
 	// Currently, implemented only for L4. This flag uses the upstream GA APIs which are not backward compatible
-	// with the advancedL4 APIs which uses a fork and a version of v1alpha1pre1
+	// with the ServicesAPI APIs which uses a fork and a version of v1alpha1pre1
```
Comment (HanFa) [inline api/v1alpha1/akodeploymentconfig_types.go:156]: is this change intended?

File: pkg/ako/values.go [hunk 2]
```diff
@@ -407,14 +407,17 @@ func DefaultControllerSettings() *ControllerSettings {
...
-func NewControllerSettings(cloudName, controllerIP, controllerVersion, serviceEngineGroup string) (setting *ControllerSettings) {
+func NewControllerSettings(cloudName, controllerIP, controllerVersion, serviceEngineGroup, tenantName string) (setting *ControllerSettings) {
 	setting = DefaultControllerSettings()
 	setting.CloudName = cloudName
 	setting.ControllerIP = controllerIP
 	setting.ServiceEngineGroupName = serviceEngineGroup
 	if controllerVersion != "" {
 		setting.ControllerVersion = controllerVersion
 	}
+	if tenantName != "" {
```
Comment (HanFa) [inline pkg/ako/values.go:418]: we don't need this if statement for now since `tenantName` is default to be empty

**Review by HanFa — APPROVED** [review]
remember to update `config/samples/network_v1alpha1_akodeploymentconfig.yaml` as well

File: api/v1alpha1/akodeploymentconfig_types.go (hunk 1 above)
Comment (XudongLiuHarold) [reply api/v1alpha1/akodeploymentconfig_types.go:156]: nah, will change it back

**Pull request merged**